salt/states/boto3_elasticache.py

```diff
@@ -87,6 +87,13 @@
             key: askdjghsdfjkghWupUjasdflkdfklgjsdfjajkghs
 """
 
+__deprecated__ = (
+    3009,
+    "Potassium",
```

Comment:
**question (blocking):** with the latest changes, the code name needs to be removed from the state modules now, correct?